packages/OmegaPrint-Core.package/OPPrinter.class/instance/literalArrayWith..st

```diff
@@ -0,0 +1,8 @@
+actions
```

Comment:
wuerde ich nicht in actions packen, weil das selbst nicht direkt zu einer Regel korrespondiert.